internal/build/build.go

```diff
@@ -187,3 +188,19 @@ func randString(n int) string {
 	}
 	return string(b)
 }
+
+func latestVersion(versions []*api.Version) *api.Version {
```

Comment:
Considering that this is used twice (for latest version, and the reverse for earliestVersion), is there any utility in changing it slightly to accept a comparison func, so you could call it with something like
```
latestFunc := func(version, latest *api.Version) bool { return version.Compare(latest) > 0 }
appropriateVersion(versions []*api.Version, latestFunc)
```
?

Reply:
Great idea! Applied suggestion.